### ISSUE TITLE
A11y Feedback: Pagination - Lack of feedback for screen readers

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1391]
CENG_1962

https://react-magma.cengage.com/version/4.5.0/api/pagination/

### Issue:
Currently when a pagination button is activated, focus remains on the button and screen reader users will hear no feedback that content on the page changed, while visual users can see that the current page was updated based on the change in color of the pagination buttons. If operating the pagination controls will not load a new page and focus will remain on the button that is activated, it is recommended to provide some type of status message so that screen reader users will feel confident that the button operated as intended.

### Suggestion:
Consider using aria-live around any content that would serve to communicate that the page updated. For example, if there is visible content above the results that indicated "page 2 or 5" then that should be identified as a live region. Otherwise this status message can be provided only to screen reader users by positioning the content offscreen. The status role can also be used in place of aria-live. See https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22 for more information.
